test/common/router/config_impl_test.cc

```diff
@@ -2744,58 +2657,9 @@ static Http::TestHeaderMapImpl genRedirectHeaders(const std::string& host, const
 }
 
 TEST_F(RouteMatcherTest, DirectResponse) {
-  static const std::string v1_json = R"EOF(
-{
-  "virtual_hosts": [
-    {
-      "name": "www2",
-      "domains": ["www.lyft.com"],
-      "require_ssl": "all",
-      "routes": [
-        {
-          "prefix": "/",
-          "cluster": "www2"
-        }
-      ]
-    },
-    {
-      "name": "api",
-      "domains": ["api.lyft.com"],
-      "require_ssl": "external_only",
-      "routes": [
-        {
-          "prefix": "/",
-          "cluster": "www2"
-        }
-      ]
-    },
-    {
-      "name": "redirect",
-      "domains": ["redirect.lyft.com"],
-      "routes": [
-        {
-          "path": "/host",
-          "host_redirect": "new.lyft.com"
-        },
-        {
-          "path": "/path",
-          "path_redirect": "/new_path"
-        },
-        {
-          "path": "/host_path",
-          "host_redirect": "new.lyft.com",
-          "path_redirect": "/new_path"
-        }
-      ]
-    }
-  ]
-}
-  )EOF";
-
   const auto pathname =
       TestEnvironment::writeStringToFileForTest("direct_response_body", "Example text 3");
 
-  // A superset of v1_json, with API v2 direct-response configuration added.
   static const std::string v2_yaml = R"EOF(
```

Comment:
minor nit but maybe this shouldn't reference v2 anymore?

Reply:
sure